src/components/Select/Select.test.tsx

```diff
@@ -0,0 +1,563 @@
+      <Select clearButtonName="clear" placeholder="라이브러리를 선택하세요">
+        <option value="react">React</option>
+        <option value="vue">Vue</option>
+      </Select>,
+    );
+
+    const clearButton = screen.queryByRole("button", {
+      name: /clear/i,
+    });
+    expect(clearButton).not.toBeInTheDocument();
+  });
+});
+
+describe("사용자 액션", () => {
+  test("옵션을 선택하면 선택된 값이 표시된다", async () => {
+    const user = userEvent.setup();
+
+    render(
+      <Select placeholder="라이브러리를 선택하세요">
+        <option value="react">React</option>
+        <option value="vue">Vue</option>
+        <option value="svelte">Svelte</option>
+      </Select>,
+    );
+
+    const select = screen.getByRole("combobox");
+    await user.selectOptions(select, "react");
+
+    expect(screen.getByText("React")).toBeInTheDocument();
+  });
+
+  test("옵션 선택 시 onChange를 호출한다", async () => {
+    const user = userEvent.setup();
+    const handleChange = vi.fn();
+
+    render(
+      <Select onChange={handleChange} placeholder="라이브러리를 선택하세요">
+        <option value="react">React</option>
+        <option value="vue">Vue</option>
+        <option value="svelte">Svelte</option>
+      </Select>,
+    );
+
+    const select = screen.getByRole("combobox");
+    await user.selectOptions(select, "react");
+
+    expect(handleChange).toHaveBeenCalledTimes(1);
+    expect(handleChange).toHaveBeenCalledWith(
+      expect.objectContaining({
+        target: expect.objectContaining({ value: "react" }),
+      }),
+    );
+  });
+
+  test("disabled가 true일 때 사용자 액션이 차단된다", async () => {
+    const handleChange = vi.fn();
+
+    render(
+      <Select
+        disabled
+        onChange={handleChange}
+        placeholder="라이브러리를 선택하세요"
+      >
+        <option value="react">React</option>
+        <option value="vue">Vue</option>
+      </Select>,
+    );
+
+    const select = screen.getByRole("combobox");
+    expect(select).toBeDisabled();
+    expect(handleChange).not.toHaveBeenCalled();
+  });
+
+  test("키보드로 옵션을 탐색할 수 있다", async () => {
+    const user = userEvent.setup();
+
+    render(
+      <Select placeholder="라이브러리를 선택하세요">
+        <option value="react">React</option>
+        <option value="vue">Vue</option>
+        <option value="svelte">Svelte</option>
+      </Select>,
+    );
+
+    const select = screen.getByRole("combobox");
+    select.focus();
+
+    await user.keyboard("{ArrowDown}");
+    await user.keyboard("{Enter}");
+
+    expect(screen.getByText("React")).toBeInTheDocument();
+  });
+
+  test("여러 옵션을 순차적으로 선택할 수 있다", async () => {
+    const user = userEvent.setup();
+
+    render(
+      <Select placeholder="라이브러리를 선택하세요">
+        <option value="react">React</option>
+        <option value="vue">Vue</option>
+        <option value="svelte">Svelte</option>
+      </Select>,
+    );
+
+    const select = screen.getByRole("combobox");
+
+    await user.selectOptions(select, "react");
+    expect(screen.getByText("React")).toBeInTheDocument();
```

Comment:
Testing Library의 Guding Principles에 따라서 가급적 `getByRole()`을 우선적으로 고려해주셨으면 합니다. 다른 곳도 마찬가지입니다.

https://testing-library.com/docs/queries/about#priority

```suggestion
    expect(screen.getByRole("option", { name: /React/ }).selected).toBe(true);
```

Reply:
option role을 생각하지 못했네요 확인해주셔서 감사합니다. 수정하겠습니다.